frontend/src/components/maps/leaflet/LayerPopup/hooks/useLayerQuery.tsx

```diff
@@ -85,17 +93,22 @@ export const saveParcelDataLayerResponse = (
   latLng?: LatLng,
 ) => {
   if (resp?.features?.length > 0) {
```

Comment:
Happy to see this finally getting implemented.

Reply:
Same here, it's been sitting dormant for a long time. :)